package/kernel/linux/modules/other.mk

```diff
@@ -727,6 +724,12 @@ define KernelPackage/zram/config
         config KERNEL_ZRAM_BACKEND_ZSTD
                 bool "zstd compression support"
 
+        config KERNEL_ZRAM_BACKEND_FORCE_LZO
+                def_bool !KERNEL_ZRAM_BACKEND_LZ4 && \
+                         !KERNEL_ZRAM_BACKEND_LZ4HC && \
+                         !KERNEL_ZRAM_BACKEND_ZSTD
```

Comment:
Maybe add also !KERNEL_ZRAM_BACKEND_LZO because when KERNEL_ZRAM_BACKEND_LZO=y then KERNEL_ZRAM_BACKEND_FORCE_LZO is unnecessary.